common/persistence/domainReplicationQueue.go

```diff
@@ -169,6 +180,76 @@ func (q *domainReplicationQueueImpl) GetAckLevels() (map[string]int, error) {
 	return q.queue.GetAckLevels()
 }
 
+func (q *domainReplicationQueueImpl) GetMessagesFromDLQ(
```

Comment:
can we add unit test for these new added functions?

Reply:
It is a little hard to add the unit test b/c the queue interface mock generation. I will try to add one